File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis.VisualBasic.Symbols/QuickAttributes.cs
using System;

namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
{
	[Flags]
	internal enum QuickAttributes : byte
	{
		None = 0,
		Extension = 1,
		Obsolete = 2,
		MyGroupCollection = 4,
		TypeIdentifier = 8
	}
}

File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis/VisualBasic/Symbols/QuickAttributes.vb
Imports System

Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
	<Flags>
	Friend Enum QuickAttributes As Byte
		None = 0
		Extension = 1
		Obsolete = 2
		MyGroupCollection = 4
		TypeIdentifier = 8
	End Enum
End Namespace